src/main/java/core/basesyntax/service/FileServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileServiceImpl implements FileService {
```

Comment:
Your FileService implementation should be placed in a service package, but it should also be in a sub-package that groups file handling classes together, for example, 'service.file'.